portlets/src/main/webapp/vue-app/realizations/components/FilterRealizationsProgramList.vue

```diff
@@ -90,7 +88,7 @@ export default {
   },
   watch: {
     selected() {
-      this.partiallySelected = this.selected.length !== this.programsList.length;
+      this.$emit('empty-list', this.emptyList);
```

Comment:
useless IMO!

Reply:
this value is going to be catched by the parent component 'FilterRealizationsDrawer' to disable the the confirm button whenever the list list is empty !